public/partials/navbar.html

```
@@ -232,19 +232,53 @@
+                                <a href="/notifications.html" class="flex items-center px-4 py-2 text-sm text-gray-700 dark:text-gray-300 hover:bg-teal-50 dark:hover:bg-teal-900 hover:text-teal-600 dark:hover:text-teal-300">
+                                    <i class="fas fa-bell mr-2 text-teal-500 w-4 text-center"></i>Notifications
+                                </a>
+                                <a href="/notification-preferences.html" class="flex items-center px-4 py-2 text-sm text-gray-700 dark:text-gray-300 hover:bg-teal-50 dark:hover:bg-teal-900 hover:text-teal-600 dark:hover:text-teal-300">
+                                    <i class="fas fa-sliders-h mr-2 text-teal-500 w-4 text-center"></i>Preferences
+                                </a>
+                            </div>
+
+                            <hr class="border-gray-200 dark:border-gray-700 my-1">
+
+                            <div class="py-1">
+                                <button onclick="logout()" class="w-full text-left flex items-center px-4 py-2 text-sm text-red-600 dark:text-red-400 hover:bg-red-50 dark:hover:bg-red-900/20">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)